src/dpr/views/dashboard.njk

```diff
@@ -0,0 +1,26 @@
+{% from "../components/chart-card/view.njk" import dprChartCard %}
+
+{% extends layoutPath %}
+{% set mainClasses = "app-container govuk-body" %}
+{% block content %}
+  {# TODO: 👇🏽 These scripts need to go into the main UI and tools 👇🏽 #}
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@4.4.4/dist/chart.umd.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/chartjs-plugin-datalabels@2.0.0"></script>
+  <script>
+    Chart.defaults.font.family = 'GDS Transport'
+    Chart.register(ChartDataLabels)
+    Chart.defaults.font.size = 16;
+  </script>
```

Comment:
Chrome will throw security warnings about an in-line script. Could this be included in the `all.mjs` file's initialisation?